src/sonic-host-services/scripts/hostcfgd

```diff
@@ -120,9 +142,9 @@ class FeatureHandler(object):
         self._config_db = config_db
         self._device_config = device_config
         self._cached_config = {}
-        self.is_multi_npu = device_info.is_multi_npu()
+        self.is_multi_npu_device = is_multi_npu()
 
-    def handle(self, feature_name, feature_cfg):
+    def handle(self, feature_name, op, feature_cfg):
```

Comment:
Suggest more descriptive name, like "operation"

Reply:
I think "op" is fine, most of the orch classes i know uses op. It conveys the meaning and is crisp